docs/src/developing/clients/jsonrpc-api.md

```diff
@@ -2125,20 +2125,21 @@ Result:
 
 ### getRecentPrioritizationFees
 
+**UNSTABLE**
 Returns a list of minimum prioritization fees from recent blocks. Currently, a
 node's prioritization-fee cache stores data from up to 150 blocks.
 
 #### Parameters:
 
-- `<array>` - (optional) An array of account address strings. If this parameter is provided, the response will reflect the minimum prioritization fee to land a transaction locking all of the provided accounts as writable.
+- `<array>` - (optional) An array of account address strings (up to a maximum of 128 addresses). If this parameter is provided, the response will reflect the minimum prioritization fee to land a transaction locking all of the provided accounts as writable.
```

Comment:
looks like we accept loading up to 256 accounts per transaction via ALT (which is probably reckless, I thought it was 64)

https://github.com/solana-labs/solana/blob/69c4db27c58ce133ff0b611bac723b992ec9d325/sdk/program/src/message/versions/v0/mod.rs#L133

Reply:
nm, it's still capped at load to 64 in `Bank::get_transaction_account_lock_limit`

Reply:
Are you suggesting we change this? https://github.com/solana-labs/solana/blob/56f8c164975a2b68762e7fe22057955bcacdefd3/rpc/src/rpc.rs#L4004

Because `MAX_TX_ACCOUNT_LOCKS` is definitely set to 128

Reply:
`MAX_TX_ACCOUNT_LOCKS` is only in use on devnet.  64 is hardcoded on the other clusters 'til the feature is activated

https://github.com/solana-labs/solana/blob/56f8c164975a2b68762e7fe22057955bcacdefd3/runtime/src/bank.rs#L3628

pretty great that we have three live values implying that they do the same thing

Reply:
> `MAX_TX_ACCOUNT_LOCKS` is only in use on devnet.

Why is that feature active on devnet and not testnet? 💔 
Well, anyway, `MAX_TX_ACCOUNT_LOCKS` is what the `getRecentPrioritizationFees` RPC is using, so this doc is correct. Let's create an issue elsewhere, if that value should be changed.